space_invaders.c

```diff
@@ -101,6 +103,14 @@ void generate_enemies(void) {
     
 }
 
+void initialize_enemy_bullet_array(void) {
+    for (uint8_t i = 0; i < ENEMY_BULLET_LIMIT; i++) {
+        enemy_bullets[i].pos.row = 0;
+        enemy_bullets[i].pos.column = 0;
+        enemy_bullets[i].active = 0;
+    }
+}
```

Comment:
All zeros? This should actually be default initialisation of a global var. But it does't hurt to _ensure_ it is. :+1: